rocketpy/sensors/gnss.py

```diff
@@ -0,0 +1,144 @@
+import math
+
+import numpy as np
+
+from ..mathutils.vector_matrix import Matrix, Vector
+from ..prints.sensors_prints import _GNSSPrints
+from .sensor import ScalarSensor
+
+
+class GNSS(ScalarSensor):
+    """Class for the GNSS sensor
+
+    Attributes
+    ----------
+    prints : _GNSSPrints
+        Object that contains the print functions for the sensor.
+    sampling_rate : float
+        Sample rate of the sensor in Hz.
+    position_accuracy : float
+        Accuracy of the sensor interpreted as the standard deviation of the
+        position in meters.
+    altitude_accuracy : float
+        Accuracy of the sensor interpreted as the standard deviation of the
+        position in meters.
+    name : str
+        The name of the sensor.
+    measurement : tuple
+        The measurement of the sensor.
+    measured_data : list
+        The stored measured data of the sensor.
+    """
+
+    units = "°, m"
+
+    def __init__(
+        self,
+        sampling_rate,
+        position_accuracy=0,
+        altitude_accuracy=0,
+        name="GNSS",
+    ):
+        """Initialize the GNSS sensor.
+
+        Parameters
+        ----------
+        sampling_rate : float
+            Sample rate of the sensor in Hz.
+        position_accuracy : float
+            Accuracy of the sensor interpreted as the standard deviation of the
+            position in meters. Default is 0.
```

Comment:
These instruments usually have different accuracy in different axes. I would expect an accuracy for z, another for x, and third one for y. And to be more accurate, we usually see errors being expressed in terms of longitude/latitude.

```suggestion
            Accuracy of the sensor interpreted as the standard deviation of the
            position in meters. Default is 0.
```

Reply:
> These instruments usually have different accuracy in different axes. I would expect an accuracy for z, another for x, and third one for y. And to be more accurate, we usually see errors being expressed in terms of longitude/latitude.

Actually, I would expect one "horizontal accuracy" and another "vertical accuracy".
This is what I'm currently familiar with. But my knowledge about specific GNSS instruments for rockets is almost zero.

All in all you can let only just a single accuracy here, it's okay. If someone else complains, we can get back to it in the future.


Example:
https://globalgis.lk/storage/2022/05/E800_Datasheet_20201111_ENG-H.pdf
![image](https://github.com/user-attachments/assets/e5016907-951d-495b-86a5-b4a24a994461)